index.html

```diff
@@ -14,11 +14,39 @@
   <link rel="stylesheet" href="css/reset.css">
   <link rel="stylesheet" href="css/layout.css">
   <link rel="stylesheet" href="css/styles.css">
+  <link href="https://fonts.googleapis.com/css2?family=Open+Sans&family=Proza+Libre:wght@400;600&display=swap"
+  rel="stylesheet">
 </head>
 <body>
   <header>
+    <div class="container">
+      <div class="row">
+        <div class="column-full">
+          <p>Code Journal</p>
+        </div>
+      </div>
+    </div>
   </header>
   <main>
+    <div data-view="entry-form" class="container">
+      <form class="row">
+        <h1 class="column-full">New Entry</h1>
+        <div class="column-full column-half display-image">
```

Comment:
I think you want just `column-half`, not `column-full` here, since `column-half` becomes full width on mobile with the styling we provided for you.